scripts/gee

```diff
@@ -4567,19 +4568,42 @@ function gee__bazelgc() {
                 ) | sort | uniq -c | awk '$1 == "1" {print $2}' )
   if [[ "${#EXPIRED[@]}" == 0 ]]; then
     _info "No ~/.cache/bazel directories to delete."
-    return 0
+  else
+    _info "About to delete the following ~/.cache/bazel directories:" "${EXPIRED[@]}"
+    if _confirm_default_no "Proceed? (y/N)  "; then
+      local USED1
+      USED1="$(df -k --output=used ~/.cache | tail -1)"
+      _cmd chmod -R u+w "${EXPIRED[@]}"
+      _cmd rm -rf "${EXPIRED[@]}"
+      USED2="$(df -k --output=used ~/.cache | tail -1)"
+      MBFREED="$(( (USED1 - USED2) / 1024 ))"
+      _info "Approximately ${MBFREED}MiB freed."
+    else
+      _info "Garbage collection of ~/.cache/bazel aborted, nothing was deleted."
+    fi
   fi
-  _info "About to delete the following ~/.cache/bazel directories:" "${EXPIRED[@]}"
-  if _confirm_default_no "Proceed? (y/N)  "; then
-    local USED1
-    USED1="$(df -k --output=used ~/.cache | tail -1)"
-    _cmd chmod -R u+w "${EXPIRED[@]}"
-    _cmd rm -rf "${EXPIRED[@]}"
-    USED2="$(df -k --output=used ~/.cache | tail -1)"
-    MBFREED="$(( (USED1 - USED2) / 1024 ))"
-    _info "Approximately ${MBFREED}MiB freed."
+
+  if [[ -d ~/.cache/bazel-disk-cache ]]; then
+    local -a OLD=()
+    mapfile -t OLD -d '' < \
+      <(find ~/.cache/bazel-disk-cache \( -type f -o -type l \) -atime +5 -print0)
+    if [[ "${#OLD[@]}" == 0 ]]; then
```

Comment:
`"${#OLD[@]}"` is 1 here. even though i have many old files. Something's going wrong with `-print0` above?

Reply:
Thanks for catching that!  It turns out the argument order for mapfile is significant, so the right invocation was `mapfile -d '' OLD`.  Fixed.